src/components/AddElement.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled from 'styled-components/native'
+
+import { AddCircleIcon } from '../../assets/images'
+import { ContentSecondary } from './text/Content'
+import { SubheadingPrimary } from './text/Subheading'
+
+const Root = styled.TouchableOpacity`
+  margin: ${props => props.theme.spacings.sm} auto;
+  padding: ${props => props.theme.spacings.xs};
+`
+
+const AddCustomDisciplineText = styled(SubheadingPrimary)`
+  text-transform: uppercase;
+`
+
+const Explanation = styled(ContentSecondary)`
+  padding: 0 ${props => props.theme.spacings.xxl} 0 15%;
```

Comment:
Yes right. I was really racking my brain here, as we need a padding that has the width of the icon plus the padding. In css i would have used grid, but this is not supported in react native, also css-calc function is not supported. Therefore I decided this is the best solution, although I am also not really happy with it. If you have a better solution, let me know. Otherwise I think it is fine to keep it.

Reply:
I definitely don't have a better solution, css and styling is not really my sahneseite :D maybe @f1sh1918 has one though